readme/7.1/BREAKING_CHANGES.markdown

```diff
@@ -452,4 +452,40 @@ environment is a very serious security issue.
 A value of `true` will improve a portal administrator's experience, and a value
 of `false` can be considered during performance tuning, if needed.
 
+---------------------------------------
+
+### Users can have numeric screen names with no limitations, and sites can no longer have numeric friendly URLs
+- **Date:** 2017-Oct-10
+- **JIRA Ticket:** LPS-66460
+
+#### What changed?
+
+- The portal property `users.screen.name.allow.numeric` now defaults to **true**
+- Numeric screen names are no longer limited by whether or not they correspond to an existing group ID.
+- Groups are no longer allowed to set their group ID as their friendly URL
```

Comment:
Should we use the word `sites` for `groups` here instead?  I think we normally use the word `sites` for `groups`  in docs since `groups` is vague and can be used to refer to different things.

Reply:
Yes, we definitely should.  Good catch.  Hopefully I was consistent with that everywhere else.